.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
See for example this:
https://travis-ci.org/diydrones/ardupilot/builds/113119166

You have a huge matrix of OS + compiler. There's no reason to compile navio, raspilot, etc with gcc (or clang). You will have most of the builds failing due to a timeout in travis.  Adding 2 more targets is ok (clang on linux for sitl, clang on macos for sitl) but duplicating the number of jobs is not.